### DISCUSSION TITLE
Welcome to dsmtpd Discussions!

### DISCUSSION THREAD

**Discussion by matrixise** [matrixise/dsmtpd#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by matrixise** [comment]
@bernhardreiter

**Reply by bernhardreiter** [reply]
Hi, 
my company uses dsmtpd for one (Free Software) product where we need to test a lot of outgoing emails during development and test setups. Details are here https://github.com/intevation/intelmq-mailgen . 

In general we love Free Software, so when we have an idea or a small improvement, we try to communicate it.

**Reply by matrixise** [reply]
and do you have some suggestions? On this morning, I wanted to add a web interface with websocket and vuejs and movet to asyncio/aiohttp/aiosmtp

**Reply by bernhardreiter** [reply]
No suggestions at this time. Currently, I do not think a webinterface is a good idea for modularity, as there are so many mailclients out there, you could just use one, like http://squirrelmail.org/download.php (very traditional. ;)) to Mailpile (a bit too modern at places >;) ).

Moving to aiosmtp would just be an exercise to make it more future proof or learn the new modules, it is not really necessary in a stricter sense.

Best Regards and thanks for asking!
